src/App.js

```diff
@@ -1,25 +1,42 @@
-import logo from './logo.svg';
 import './App.css';
+import foodsData from './foods.json';
+import { Row, Divider } from 'antd';
+import { useState } from 'react';
+
+import FoodBox from './Components/FoodBox';
+import AddFoodForm from './Components/AddFoodForm';
+import Search from './Components/Search';
 
 function App() {
+  const [foods, setFoods] = useState(foodsData);
+  const [filteredFoods, setFilteredFoods] = useState(foodsData);
+
+  const addNewFood = (newFood) => {
+    const updatedFoods = [...foods, newFood];
+    setFoods(updatedFoods);
```

Comment:
When you create a new food on the website nothing appears, and the function to `addNewFood` is correct. The only thing that you are missing is that you also need to update the `setFilteredFoods()` with the `updatedFoods` that you created, because that's the one that you are using the `.map` method to render the list of all the foods. 
The way you did, you are only updating the `foods` array, and not the filtered array that you are showing to the user.